### ISSUE TITLE
PYTHON

### ISSUE THREAD

**Issue by mesquitaagro** [tihanyin/PSSW100AVB#9]
If it were in python it would be more practical.

**Comment by tihanyin** [comment]
Indeed, Python is highly valuable for writing exploits and is widely used within the penetration testing community. While PowerShell scripts can easily be converted to Python, the latter is not natively included in the Windows ecosystem like PowerShell. This repository is intentionally focused on PowerShell.
